appconfig/appconfig.go

```diff
@@ -92,6 +119,26 @@ func Init() error {
 	return nil
 }
 
+func InitQueryWriter(globalLogsWriter io.Writer) (io.Writer, error) {
```

Comment:
I usually use Init() method if the method doesn't return the created value, but sets it to global variable (like log).
If a method returns the created value - it's name is New..()

Reply:
Fixed